src/Model/PaymentMethod.php

```diff
@@ -52,12 +58,11 @@ public function setBillingAddress(Address $address): self
 
     public function toArray(): array
     {
-        if (!isset($this->billingAddress)) {
-            return [];
-        }
-
+        // If the last4, name, and type are added to this, then they
+        // will need to be stripped off prior to it being submitted to
+        // Braintree as part of a subscription in PaymentMethodMapper::toRequest.
         return [
-            'billingAddress' => $this->billingAddress->toArray(),
+            'billingAddress' => $this->billingAddress?->toArray(),
```

Comment:
this is a slight behavioral change... where before an unset billing address would return `[]` but now it would return `['billingAddress' => null'];`

Should we revert the change, but keep the comment?

Reply:
Yes, I had removed that when I was originally packing all the fields in the `toArray`, but I've reverted it back to what it was originally since I am not doing that anymore.  Thanks for catching that. ❤️